ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -555,16 +893,35 @@ def _useRealization(self, info, rlz):
       else:
         children = childrenMutated
 
+      # # Make sure no children are exactly similar to parents
+      # flag = True
+      # counter = 0
+      # while flag and counter < self._populationSize:
+      #   counter += 1
+      #   repeated =[]
+      #   for i in range(np.shape(self.population.data)[0]):
+      #     for j in range(i,np.shape(children.data)[0]):
+      #       if all(self.population.data[i,:]==children.data[j,:]):
+      #         repeated.append(j)
+      #         repeated = list(set(repeated))
+      #         if repeated:
+      #           newChildren = self._mutationInstance(offSprings=children[repeated,:],
+      #                                                distDict=self.distDict,
+      #                                                locs=self._mutationLocs,
+      #                                                mutationProb=self._mutationProb,
+      #                                                variables=list(self.toBeSampled))
+      #           children.data[repeated,:] = newChildren.data
+      #         else:
+      #           flag = False
```

Comment:
remove?

Reply:
We need to re-write it, adding this back as an option will be a further pull request probably, better code is in https://github.com/Jimmy-INL/raven/commits/testing-duplicant-removal/ so I will remove the commented out code from here.

Reply:
Fixed in 1ac6ede0bb89ab27658859652f644aeddeb8a3ab